contrib/babelfishpg_common/src/datetime.h

```diff
@@ -32,6 +32,8 @@
 #define END_DATETIME	INT64CONST(252455615999999000)
 /* TSQL default datetime: 1900-01-01 00:00:00.000 */
 #define TSQL_DEFAULT_DATETIME	INT64CONST(-3155673600000000)
+/* minimun value for datepart general_integer_datatype */
+#define DATEPART_MIN_VALUE	INT64CONST(-53690)
```

Comment:
Lets add the date and time corresponding to this constant value in the comment, as done for constant `TSQL_DEFAULT_DATETIME`.